scripts/genesis2db/main.go

```diff
@@ -85,23 +87,23 @@ func main() {
 
 	i = 0
 	logrus.Info("import followings")
-	for follower, v := range g.AppState.Community.Followers {
```

Comment:
Did you rename Followers to Following? Is it not a mistake?

Reply:
yeah, I think it transfers the meaning more precise